### DISCUSSION TITLE
Can I have my task?

### DISCUSSION THREAD

**Discussion by Huangtianyu-caas** [open-mmlab/mmdetection3d#2158] [General]
Hello,

I would like to have my task.(huangtianyu-MMDetection3D-46)
Thank you!

Best regards
Huang Tianyu

**Comment by JingweiZhang12** [comment]
@Huangtianyu-caas . Welcome. Please leave a comment below https://github.com/open-mmlab/mmdetection3d/discussions/1982 and take a specific task.
